modules/tinymce/changelog.txt

```diff
@@ -1,5 +1,5 @@
 Version 5.6.0 (TBD)
-
+    Fixed keyword font sizes such as `medium` not displaying correctly in font select menus #TINY-6291
```

Comment:
```suggestion
    Fixed keyword font sizes such as `medium` not displaying correctly in font size menus #TINY-6291
```
I got confused at first because I thought you meant the `fontselect` (since that's an actual thing). I think this is clearer? Though maybe it should be even more specific and reference `fontsizes` menu and `fontsizeselect` toolbar button? Not sure. Hopefully @tylerkelly13 will have thoughts.